ci/docker/debug-cuda-gh200-scalapack.yaml

```diff
@@ -35,6 +35,5 @@ spack:
         - '+fortran'
     pika:
       require:
-        - '@0.30.1'
```

Comment:
Should be set to 0.32.0 as to keep a CI pipeline with an older pika version when we upgrade.

Reply:
Hmm, good point. Though given that my current hypothesis that pika 0.32.0 and 0.30.1 are equally good/bad, should we just keep this configuration unchanged then? And keep the threads check disabled? In that case I can close this PR...

Reply:
I would prefer to have thread checks enabled for all pipelines.
(Otherwise we might forget to re-enable them in the future)

Reply:
Fair enough! I fixed it to 0.32.0 now, and reenabled the threads check.